File: javascript/gemini/p124_nre.js
const args = process.argv.slice(2);

if (args.length !== 1) {
    console.error("Error: Exactly one argument required.");
    process.exit(1);
}

const input = args[0];
console.log(input);
const trimmed = input.trim();
console.log(trimmed);
process.exit(0);

File: java/gemini/p124_nre.java
public class p124_nre {
    public static void main(String[] args) {
        if (args.length != 1) {
            System.err.println("Error: Exactly one argument required.");
            System.exit(1);
        }

        String input = args[0];
        System.out.println(input);
        String trimmed = input.trim();
        System.out.println(trimmed);
        System.exit(0);
    }
}